client/src/App.tsx

```diff
@@ -32,6 +34,12 @@ function App(): JSX.Element {
                   <Route path="/sitter/:profileId">
                     <ProfileDetails />
                   </Route>
+                  <Route exact path="/myprofile">
+                    <MyProfile />
+                  </Route>
+                  <Route exact path="/myprofile/editphoto">
```

Comment:
Nit: We should edit the routing here to be a bit more conventional and clean like `/profile/edit/photo` 

Also, what are your thoughts on moving this into the profile settings?